### ISSUE TITLE
[Bug]: jio tv go stopped working

### ISSUE THREAD

**Issue by Dhruvpatel2112** [siddharthsky/CustTermux-JioTVGo#54]
### Problem description

otp authentication isn't woking suddenly.

### What steps will reproduce the bug?

otp authentication isn't workin.

### System information

```shell
using latest version,
```

**Comment by Sunmayj** [comment]
Same hear @siddharthsky . App is not working on any device. Screen shot attached. Status has error. I also tried to reinstall from extra but still same issue. 
![IMG20240917172119](https://github.com/user-attachments/assets/7b93b438-034c-47b5-8424-1a639693d1f5)

**Comment by 888594** [comment]
that was because of jio server went down. check with original jiotv app first.

**Comment by Sunmayj** [comment]
Yes mobile app was fine. Looks like some users and network type were impacted 
 This could be due to the data centre outage. It's now working fine now.
Outage news. 
https://timesofindia.indiatimes.com/technology/tech-news/reliance-jio-confirms-outage-heres-what-the-company-said/articleshow/113425602.cms
